src/network/network_content.cpp

```diff
@@ -82,7 +82,7 @@ static HasContentProc *GetHasContentProcforContentType(ContentType type)
 
 bool ClientNetworkContentSocketHandler::Receive_SERVER_INFO(Packet &p)
 {
-	ContentInfo *ci = new ContentInfo();
+	auto ci = std::make_unique<ContentInfo>();
```

Comment:
why does this need to be a pointer? i.e. make `infos` a `std::vector<ContentInfo>` ?

Reply:
> Smart pointers are used instead of a straight vector as the network content window maintains its own sorted list of content which references each element.